packages/docs/docs/charting/allergy-intolerances.md

```diff
@@ -0,0 +1,81 @@
+# Representing Allergies
+
+Allergies are recorded via [`AllergyIntolerance`](/docs/api/fhir/resources/allergyintolerance). Some relevant fields are described below: 
+
+| Element                    | Description                                                                                            |
+| -------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `clinicalStatus`           | The status of the allergy. Allergies may become inactive or re-activate over time.                     |
+| `verificationStatus`       | Whether the allergy has been confirmed by the clinician.                                               |
+| `category`                 | Whether the allergy is food, medication, environmental, or biologic.                                   |
+| `code`                     | The code, along with the code system. In the US, this is commonly RxNorm (medications) or SNOMED (food + environment).                              |
+| `patient`                  | Patient who has this allergy.                                                                          |
+| `encounter`                | The Encounter where this allergy was discussed.                                                        |
+| `recorder`                 | Who recorded this allergy.                                                                             |
+| `asserter`                 | Who reported this allergy (i.e. patient, parent, clinician after reviewing allergy tests, etc.)        |
+
+
+# Recording Unknown Allergy Status vs. No Known Allergies 
+
+In healthcare, there is a distinction between unknown allergy status (no recorded allergies for the patient, perhaps because no one had asked), and the patient having confirmed that they have no allergies. 
```

Comment:
These recommendations are all correct, but I think we need to structure this content to more "hamburger" to help people index this. 

At the top of this section, put together a checklist of the inputs to the modeling decision. For example: 
* Clinical verificatoin: Confirmed vs. unconfirmed 
* Positive vs. negative assertion 
* Clinically relevant? 


Then address each of those in an h3 section with a description and an example.

Reply:
Note that the "negated" condition/allergy is a tricky one for which there is no consensus
* https://hl7.org/fhir/R4/allergyintolerance.html#9.1.3.3
* https://chat.fhir.org/#narrow/channel/179166-implementers/topic/.22Negative.20knowledge.22.20implementations/near/210935451
* https://chat.fhir.org/#narrow/channel/179166-implementers/topic/negated.20Condition

For now, we might want to go with the idea of using the `verificationStatus=refuted` for negating specific allergies. The gap in this guidance is the following situation: 
- Patient say's their not allergic to peanuts, but doctor has not confirmed (`verificationStatus=unknown`)
- Later on the doctor confirms the absence of allergy via blood test  (`verificationStatus=refuted`)